dashboard/app/models/section.rb

```diff
@@ -40,6 +40,11 @@ class Section < ActiveRecord::Base
   LOGIN_TYPE_PICTURE = 'picture'
   LOGIN_TYPE_WORD = 'word'
 
+  TYPES = [
+    TYPE_PD_WORKSHOP = 'pd_workshop'
```

Comment:
s/TYPE_PD_WORKSHOP/PD_WORKSHOP

The name is already in an array called `TYPES`...